src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -1404,7 +1404,7 @@ namespace winrt::TerminalApp::implementation
         {
             if (const auto activePane{ activeTab->GetActivePane() })
             {
-                _restartPaneConnection(activePane);
+                _restartPaneConnection(activePane->GetContent().try_as<TerminalApp::TerminalPaneContent>(), nullptr);

```

Comment:
`_restartPaneConnection` doesn't protect against nullptr, does it?

Reply:
It also doesn't use the args. We're just bubbling the TypedEvent up, but we never raised it with args in the first place. (kinda like how we don't really use `sender` in most places)